### PR TITLE
KOGITO-2856 Extract version parsing

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
@Ginxo tests are passing :)

File: vars/util.groovy [hunk 1]
```diff
@@ -211,22 +211,36 @@ GIT INFORMATION REPORT
...
+        int majorVersion = versionSplit[0] + (type == 'major' ? 1 : 0)
+        int minorVersion = versionSplit[1] + (type == 'minor' ? 1 : 0)
+        int microVersion = versionSplit[2] + (type == 'micro' ? 1 : 0)
+        return "${majorVersion}.${minorVersion}.${microVersion}${suffix ? '-' + suffix : ''}"
+    } else {
+        return null
+    }
+}
+
+Integer[] parseVersion(String version){
```
Comment (Ginxo) [inline vars/util.groovy:225]: can you please add comments about what the method is doing. Thanks.

Comment (radtriste) [reply vars/util.groovy:232]: updated. let me know

**Comment by radtriste** [comment]
@Ginxo good to merge ?

**Pull request merged**